<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta content="Jim Robinson" name="author">
    <link href=img/favicon.ico rel="shortcut icon">
    <title>positive control  </title>


</head>

<body>

<h3> Augustus gff positive control : C Elegans  </h3>




<div id="igvDiv" style="padding-top: 20px;padding-bottom: 20px; height: auto"></div>



<div id="igvDiv2" style="padding-top: 20px;padding-bottom: 20px; height: auto"></div>

<script type="module">

    import igv from "https://genome321.s3.amazonaws.com/igv.esm.min.js"

    // BEDPE examples
    const options =
        {
   			
			locus: "chrI:56,366-672,302",
            genome: "ce11",           
			
            tracks: [
			

 				{
				   
                    url: "/celegans.ch1.gff3",
                    type: "annotation",
                    format: "gff3",                 
                    name:"Augustus",
                    color: "#A9A9A9"
					
				},
				
				


            ]
        }

    igv.createBrowser(document.getElementById("igvDiv"), options)

    // UCSC interact format examples
    const options2 =
        {
            reference: {
                "id": "wclumterr",
                "name": "lumbricus terrestris",
                "fastaURL": "https://genome321.s3.amazonaws.com/sequence.fasta",
                "indexURL": "https://genome321.s3.amazonaws.com/sequence.fasta.fai",
                "cytobandURL": "https://genome321.s3.amazonaws.com/cytoBand.txt"
            },
            locus: "OX457036.1:51,512,582-51,530,885",
            tracks: [
                {
					name: 'rna-seq',
					url: 'https://genome321.s3.amazonaws.com/transcripts.gff3',
					//color: '#03A9F4',
					color:'navy',
					type: 'annotation',
					format: 'gff3'
                },
				{
					url: 'https://genome321.s3.amazonaws.com/junctionsLumTerr.bed', 
					type: 'junction',
					name: 'Junctions',
					format: 'bed',
					displayMode: 'COLLAPSED',
					minUniquelyMappedReads: 1,
					minTotalReads: 1,
					maxFractionMultiMappedReads: 1,
					minSplicedAlignmentOverhang: 0,
					colorBy: 'isAnnotatedJunction'					
				},
				
				{
					name: 'protocol1',
					url: 'https://genome321.s3.amazonaws.com/protocol1OX457036.gff',            
					format: 'gff3',
					colorBy: "biotype",
					colorTable: {
					"antisense": "blueviolet",
					"protein_coding": "blue",
					"processed_transcript": "purple",
					"processed_pseudogene": "#7fff00",
					 "*": "DimGray",
					 "unprocessed_pseudogene": "#d2691e" }
				},
				{
					name: "Protocol2",
					format: "gff3",
					displayMode: "expanded",
					height: 100,
					url: 'https://genome321.s3.amazonaws.com/protocol2OX457036.gff',                
					visibilityWindow: 1000000,
					colorBy: "biotype",
					// Uncomment to explicitly set a colorTable
					colorTable: {
					"antisense": "blueviolet",
					"protein_coding": "blue",
					"retained_intron": "rgb(0, 150, 150)",
					"processed_transcript": "purple",
					"processed_pseudogene": "#7fff00",
					"unprocessed_pseudogene": "#d2691e",
					"*": "black"}				
				
				},		
				
				{
					name: "Hintsp",
					format: "gff3",
					displayMode: "expanded",
					height: 100,
					url: 'https://genome321.s3.amazonaws.com/hintp.gff3',                
					visibilityWindow: 1000000,					
					colorBy: "REGIONID",
					// Uncomment to explicitly set a colorTable
					colorTable: {
					"antisense": "blueviolet",
					"protein_coding": "blue",
					"retained_intron": "rgb(0, 150, 150)",
					"processed_transcript": "purple",
					"processed_pseudogene": "#7fff00",
					"unprocessed_pseudogene": "#d2691e",
					"*": "red"}				
				},

            ]
        }
    igv.createBrowser(document.getElementById("igvDiv2"), options2)


</script>




</body>

</html>
